hudi-hive/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -68,6 +68,10 @@
   @Parameter(names = {"--use-jdbc"}, description = "Hive jdbc connect url")
   public Boolean useJdbc = true;
 
+  @Parameter(names = "--use-hive-style-partitioning", description = "Use Hive style partitioning, the name of "
+      + "partition folders follow <partition_column_name>=<partition_value> format")
+  public Boolean useHiveStylePartitioning = false;
```

Comment:
@zhedoubushishi : Is this still being used ? If not, Can you remove it

Reply:
Oh its not. All related code is removed now.